src/content/peerconnection/webaudio-input/js/main.js

```diff
@@ -68,11 +68,11 @@ function gotStream(stream) {
     var filteredStream = webAudio.applyFilter(stream);
 
     var servers = null;
-
-    pc1 = new webkitRTCPeerConnection(servers); // jscs:ignore requireCapitalizedConstructors
+    /*eslint new-cap: 0*/
```

Comment:
I'm curious, does this turn it off for the whole file or is it smart and registers the offending Constructor and ignore all of them?